input/postgres/relation_stats.go

```diff
@@ -65,6 +66,44 @@ SELECT c.oid,
 			 AND c.oid NOT IN (SELECT pd.objid FROM pg_catalog.pg_depend pd WHERE pd.deptype = 'e' AND pd.classid = 'pg_catalog.pg_class'::regclass)
 			 AND n.nspname NOT IN ('pg_catalog','pg_toast','information_schema')
 			 AND ($1 = '' OR (n.nspname || '.' || c.relname) !~* $1)
+ UNION
+SELECT relid,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   NULL,
+	   NULL,
+	   NULL,
+	   NULL,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   0,
+	   true AS exclusively_locked
+  FROM locked_relids_with_parents
```

Comment:
is there any case that `locked_relids_with_parents` can potentially include some relids that are not meeting the above condition? (e.g. relkind is not IN ('r','v','m','p'), IOW the lock was held with a TOAST (relkind = t))

Reply:
uhm, I guess it's okay to include these ones even, as we won't send these stats anyways 🤔

Reply:
Yeah, it definitely could pull in relids that are otherwise ignored - as you note that is not a problem though because we wouldn't send it. I think its okay to keep the query simple like that for now.